lib/stencil-download.utils.spec.js

```diff
@@ -24,31 +26,38 @@ describe('ThemeDownloader', function () {
 
         jest.spyOn(console, 'log').mockImplementation(jest.fn());
 
-        jest.spyOn(request, 'Request').mockImplementation(requestStub);
+        fetch.mockImplementation(async() => ({
```

Comment:
@jairo-bc because we are already using fetch in our other libraries, so I decided to use it here as well so that we don't need to know APIs of different libraries used for the same purposes.
But we can discuss it with other team members and choose the right library.

@junedkazi what do you think?

Reply:
There are in NPM mocks for fetch too, e.g. https://www.npmjs.com/package/fetch-mock
Here was a simple case, so I created a manual mock, but we could use advanced mocks from npm as well